apps/ejabberd/src/mongoose_commands.erl

```diff
@@ -133,6 +138,8 @@
                   | {argspec()} % a tuple of a few args (can be of any size)
                   | [typedef()]. % a list, but one element
 
+-type optargspec() :: {atom(), typedef(), term()}. % name, type, default value
+
 -type security() :: [admin | user]. %% later acl option will be added
 
 -type errortype() :: denied | not_implemented | type_error | internal. %% we should agree on a set of atoms so that the
```

Comment:
According to **Elvis**:
> Line 145 is too long: -type errortype() :: denied | not_implemented | type_error | internal. %% we should agree on a set of atoms so that the.